src/test_driver/mbed/integration_tests/common/utils.py

```diff
@@ -207,10 +207,9 @@ def connect_device_over_ble(devCtrl, discriminator, pinCode, nodeId=None):
     if nodeId == None:
         nodeId = random.randint(1, 1000000)
 
-    try:
-        devCtrl.ConnectBLE(int(discriminator), int(pinCode), int(nodeId))
-    except exceptions.ChipStackException as ex:
-        log.error("Connect device over BLE failed: {}".format(str(ex)))
+    res, err = devCtrl.ConnectBLE(int(discriminator), int(pinCode), int(nodeId))
```

Comment:
There may be documentation (`README.md`) and `.ipynb` that need updates since there are embedded examples in some.

Reply:
Adopted your suggestion, no changes here